### PR TITLE
feat: Migrate sample and sample tests - Batch 5 of 5

### LINKED ISSUE #9: Add samples and sample tests
**Describe the solution you'd like**
Please add relevant code samples and tests.

### PR DESCRIPTION
**Pull request by vinnysenthil**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #9 (Batch 5 of 5) 🦕

### CODE REVIEW

File: samples/list_model_evaluation_slices_sample.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from google.cloud import aiplatform
+from google.protobuf import json_format
+
+
+def list_model_evaluation_slices_sample(
+    project: str, model_id: str, evaluation_id: str
+):
+    client_options = {"api_endpoint": "us-central1-aiplatform.googleapis.com"}
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
```
Comment (busunkim96) [inline samples/list_model_evaluation_slices_sample.py:25]:
```suggestion
    # This client only needs to be created once, and should be reused for multiple requests.
```

File: samples/list_model_evaluation_slices_sample.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    client = aiplatform.ModelServiceClient(client_options=client_options)
+    location = "us-central1"
+    parent = "projects/{project}/locations/{location}/models/{model}/evaluations/{evaluation}".format(
+        project=project, location=location, model=model_id, evaluation=evaluation_id
+    )
```
Comment (busunkim96) [inline samples/list_model_evaluation_slices_sample.py:30]:
```suggestion
    parent = f"projects/{project}/locations/{location}/models/{model_id}/evaluations/{evaluation_id}"
```

File: samples/list_model_evaluation_slices_sample.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    response = client.list_model_evaluation_slices(parent=parent)
+    for model_evaluation_slice in response:
+        print("model_evaluation_slice")
+        print(" name:", model_evaluation_slice.name)
+        print(" metrics_schema_uri:", model_evaluation_slice.metrics_schema_uri)
+        print(
+            " metrics:", json_format.MessageToDict(model_evaluation_slice._pb.metrics)
```
Comment (busunkim96) [inline samples/list_model_evaluation_slices_sample.py:37]: @software-dov Is this the recommended way to print out the contents of a `struct.Value` on a message? https://github.com/googleapis/python-aiplatform/blob/4ddc426a6b4b8cd319fa885e363c94b35ef777d9/google/cloud/aiplatform_v1beta1/types/model_evaluation_slice.py#L30-L56

File: samples/predict_image_classification_sample.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform.PredictionServiceClient(client_options=client_options)
+    location = "us-central1"
+    name = "projects/{project}/locations/{location}/endpoints/{endpoint}".format(
+        project=project, location=location, endpoint=endpoint_id
+    )
+    # See gs://google-cloud-aiplatform/schema/predict/params/image_classification.yaml for the format of the parameters.
+    parameters_dict = {"confidence_threshold": 0.5, "max_predictions": 5}
+    parameters = json_format.ParseDict(parameters_dict, Value())
```
Comment (busunkim96) [inline samples/predict_image_classification_sample.py:35]: You can avoid calling `json_format.ParseDict` by creating a request object and passing it to the method.
```py
request = aiplatform.PredictRequest(endpoint=name)

request.parameters = {"key": "value"}) # struct.Value
request.instances.extend([{"e": "f"}, {"g": "h"}]) # Sequence[struct.Value]
request.instances.append({"l": "m"}]) # Sequence[struct.Value]

client.predict(request=request)
```

Note that for _lists of structs_ you can't assign directly (the marshaling will get confused).
```py
>>> request.instances = [{"e": "f"}, {"g": "h"}]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/google/home/busunkim/.pyenv/versions/3.8.3/lib/python3.8/site-packages/proto/message.py", line 538, in __setattr__
    self._pb.MergeFrom(self._meta.pb(**{key: pb_value}))
TypeError: Value must be iterable
```

https://github.com/googleapis/proto-plus-python/issues/104 for more details

File: samples/tests/predict_text_entity_extraction_test.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+PROJECT_ID = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+
+INSTANCE = {"content": "I really love working at Google!"}
+
+
+def test_ucaip_generated_predict_text_entity_extraction_sample(capsys):
+
+    predict_tabular_classification_sample.predict_tabular_classification_sample(
+        instance_dict=INSTANCE, project=PROJECT_ID, endpoint_id=ENDPOINT_ID
+    )
```
Comment (munkhuushmgl) [inline samples/tests/predict_text_entity_extraction_test.py:30]: Why does it use tabular_class in text_entity_extraction?

Comment (vinnysenthil) [reply samples/tests/predict_text_entity_extraction_test.py:30]: Nice, catch I'll update that now

File: samples/list_model_evaluation_slices_sample.py (hunk 2 above)
Comment (vinnysenthil) [reply samples/list_model_evaluation_slices_sample.py]: All `.format` has been replaced with f-strings

File: samples/predict_image_classification_sample.py (hunk 4 above)
Comment (vinnysenthil) [reply samples/predict_image_classification_sample.py]: Paging @dizcology for thoughts

Comment (aribray) [reply samples/predict_image_classification_sample.py]: When I wrote the Doc AI samples, I created request objects, so I've replicated that here

File: samples/list_model_evaluation_slices_sample.py (hunk 3 above)
Comment (aribray) [reply samples/list_model_evaluation_slices_sample.py]: It looks like this code was removed

**Review by dizcology — APPROVED** [review]

**Pull request merged**